sklearn/ensemble/tests/test_forest.py

```diff
@@ -373,6 +378,136 @@ def test_unfitted_feature_importances(name):
         getattr(FOREST_ESTIMATORS[name](), 'feature_importances_')
 
 
+@pytest.mark.parametrize("name", FOREST_CLASSIFIERS_REGRESSORS)
+def test_oob_importances(name):
```

Comment:
I would separate the classification and regression and have a dedicated set for each case.
I think that you can generate the data in the test itself. It is not too costly to do so.